cli.js

```diff
@@ -60,8 +55,8 @@ const scriptArgs = process.argv.slice(3)
   .join(' ')
 
 searchForScript(__dirname, scriptPath)
-  .then(
-    foundPath => executeScript(foundPath, scriptArgs),
-    triedPaths => throwSearchFail(scriptPath, triedPaths)
-  )
-  .catch(() => process.exit(1))
+  .then(foundPath => executeScript(foundPath, scriptArgs))
+  .catch(e => {
+    console.error(e.toString())
+    process.exit(1)
+  })
```

Comment:
Fixed exit code issue here as a freebee.  `.catch` is sugar for `.then(null, fn)`.  This was swallowing the error in `triedPaths => throwSearchFail(scriptPath, triedPaths)`, now we only have one catch.  Anything in the chain that throws will be caught, logged, and we'll exit correctly.